packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/template.success.tsx

```diff
@@ -77,16 +69,12 @@ const TabsContainer = styled.div`
 const Success: React.FC<Props> = (props) => {
   const [transferType, setTransferType] = useState(TransferType.DOMESTIC)
 
-  const formatIbanAddress = (): string => {
-    const EVERY_FOUR_CHARS = /(.{4})(?!$)/g
-    return props.account.address.replace(EVERY_FOUR_CHARS, `$1 `)
+  // TODO: Improve currency layouts logic
```

Comment:
Are you planning to resolve this TODO in next PR? if not you should create a jira task with a more specific description in what to do

Reply:
I'm not sure if I solved it with good practices, if you have a feedback about how I resolved it please feel free to tell me any suggestion to fix it before we send this functionality to production. If not I will remove the TODO comment 😝